firestore/integration_test_internal/src/includes_test.cc

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #include <string>
 
 #include "app_framework.h"
```

Comment:
Lint warning: `Include the directory when naming .h files`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/integration_test_internal/src/includes_test.cc:5:  Include the directory when naming .h files  [build/include_subdir] [4]"></hidden>